NVIM_APPNAME supports relative paths (#25233)
neovim/neovim: runtime/doc/news.txt

• |vim.lsp.util.locations_to_items()| sets the `user_data` of each item to the
  original LSP `Location` or `LocationLink`.

> • |$NVIM_APPNAME| environment variable now supports being set to a relative path

Reviewer: ```suggestion
• |$NVIM_APPNAME| can be set to a relative path instead of only a name.
```
